<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageDlg</class>
 <widget class="QDialog" name="ImageDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>559</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox 
{
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #E0E0E0, stop: 1 #FFFFFF);
	border: 2px solid gray;
	border-radius: 8px;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>462</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="gbNavigation">
       <property name="minimumSize">
        <size>
         <width>551</width>
         <height>68</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>551</width>
         <height>68</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QToolButton" name="tbZoomOut">
        <property name="geometry">
         <rect>
          <x>65</x>
          <y>10</y>
          <width>48</width>
          <height>48</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton
{
        border-image: url(:/imageviewerdlg/Icon/imageviewerdlg/zoom-out.png);
	background:transparent;
	border: 2px solid lightgray;
	border-radius: 10px;
}

QToolButton::hover
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,  
                                 stop: 0 #758385, stop: 0.5 #122C39,  
                                 stop: 1.0 #0E7788);  

    border-color: #11505C;
}

QToolButton::pressed
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,  
                                 stop: 0 #969B9C, stop: 0.5 #16354B,  
                                 stop: 1.0 #244F76);

	border-color: #11505C;
}

QToolTip
{
	background: #FFFFA0;
	border: 0px solid green;
    border-radius: 0px;
    padding: 0px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/icon/zoom_out2.png</normaloff>:/icon/zoom_out2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
       <widget class="QToolButton" name="tbZoomIn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>48</width>
          <height>48</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton
{
        border-image: url(:/imageviewerdlg/Icon/imageviewerdlg/zoom-in.png);
	background:transparent;
	border: 2px solid lightgray;
	border-radius: 10px;
}

QToolButton::hover
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,  
                                 stop: 0 #758385, stop: 0.5 #122C39,  
                                 stop: 1.0 #0E7788);  

    border-color: #11505C;
}

QToolButton::pressed
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,  
                                 stop: 0 #969B9C, stop: 0.5 #16354B,  
                                 stop: 1.0 #244F76);

	border-color: #11505C;
}

QToolTip
{
	background: #FFFFA0;
	border: 0px solid green;
    border-radius: 0px;
    padding: 0px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/icon/zoom_in2.png</normaloff>:/icon/zoom_in2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
       <widget class="QToolButton" name="tbNavigationPrevious">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>10</y>
          <width>48</width>
          <height>48</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton
{
        border-image: url(:/imageviewerdlg/Icon/imageviewerdlg/navigation-left.png);
	background:transparent;
	border: 2px solid lightgray;
	border-radius: 10px;
}

QToolButton::hover
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,  
                                 stop: 0 #758385, stop: 0.5 #122C39,  
                                 stop: 1.0 #0E7788);  

    border-color: #11505C;
}

QToolButton::pressed
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,  
                                 stop: 0 #969B9C, stop: 0.5 #16354B,  
                                 stop: 1.0 #244F76);

	border-color: #11505C;
}

QToolTip
{
	background: #FFFFA0;
	border: 0px solid green;
    border-radius: 0px;
    padding: 0px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/icon/zoom_pre.png</normaloff>:/icon/zoom_pre.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
       <widget class="QToolButton" name="tbNavigationNext">
        <property name="geometry">
         <rect>
          <x>305</x>
          <y>10</y>
          <width>48</width>
          <height>48</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton
{
        border-image: url(:/imageviewerdlg/Icon/imageviewerdlg/navigation-right.png);
	background:transparent;
	border: 2px solid lightgray;
	border-radius: 10px;
}

QToolButton::hover
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,  
                                 stop: 0 #758385, stop: 0.5 #122C39,  
                                 stop: 1.0 #0E7788);  

    border-color: #11505C;
}

QToolButton::pressed
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,  
                                 stop: 0 #969B9C, stop: 0.5 #16354B,  
                                 stop: 1.0 #244F76);

	border-color: #11505C;
}

QToolTip
{
	background: #FFFFA0;
	border: 0px solid green;
    border-radius: 0px;
    padding: 0px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/icon/zoom_next.png</normaloff>:/icon/zoom_next.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
       <widget class="QToolButton" name="tbRotateLeft">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>10</y>
          <width>48</width>
          <height>48</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton
{
        border-image: url(:/imageviewerdlg/Icon/imageviewerdlg/rotate-ccw.png);
	background:transparent;
	border: 2px solid lightgray;
	border-radius: 10px;
}

QToolButton::hover
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,  
                                 stop: 0 #758385, stop: 0.5 #122C39,  
                                 stop: 1.0 #0E7788);  

    border-color: #11505C;
}

QToolButton::pressed
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,  
                                 stop: 0 #969B9C, stop: 0.5 #16354B,  
                                 stop: 1.0 #244F76);

	border-color: #11505C;
}

QToolTip
{
	background: #FFFFA0;
	border: 0px solid green;
    border-radius: 0px;
    padding: 0px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/icon/zoom_roateleft.png.png</normaloff>:/icon/zoom_roateleft.png.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
       <widget class="QToolButton" name="tbRotateRight">
        <property name="geometry">
         <rect>
          <x>425</x>
          <y>10</y>
          <width>48</width>
          <height>48</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton
{
        border-image: url(:/imageviewerdlg/Icon/imageviewerdlg/rotate-cw.png);
	background:transparent;
	border: 2px solid lightgray;
	border-radius: 10px;
}

QToolButton::hover
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,  
                                 stop: 0 #758385, stop: 0.5 #122C39,  
                                 stop: 1.0 #0E7788);  

    border-color: #11505C;
}

QToolButton::pressed
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,  
                                 stop: 0 #969B9C, stop: 0.5 #16354B,  
                                 stop: 1.0 #244F76);

	border-color: #11505C;
}

QToolTip
{
	background: #FFFFA0;
	border: 0px solid green;
    border-radius: 0px;
    padding: 0px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/icon/zoom_roateright.png</normaloff>:/icon/zoom_roateright.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
       <widget class="QToolButton" name="tbFullScreenImage">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>10</y>
          <width>48</width>
          <height>48</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton
{
        border-image: url(:/imageviewerdlg/Icon/imageviewerdlg/delete.png);
	background:transparent;
	border: 2px solid lightgray;
	border-radius: 10px;
}

QToolButton::hover
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,  
                                 stop: 0 #758385, stop: 0.5 #122C39,  
                                 stop: 1.0 #0E7788);  

    border-color: #11505C;
}

QToolButton::pressed
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,  
                                 stop: 0 #969B9C, stop: 0.5 #16354B,  
                                 stop: 1.0 #244F76);

	border-color: #11505C;
}

QToolTip
{
	background: #FFFFA0;
	border: 0px solid green;
    border-radius: 0px;
    padding: 0px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/icon/zoom_fullscreen.png</normaloff>:/icon/zoom_fullscreen.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
       <widget class="Line" name="lnDelimiter4">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>10</y>
          <width>2</width>
          <height>48</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="Line" name="lnDelimiter1">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>10</y>
          <width>2</width>
          <height>48</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="Line" name="lnDelimiter3">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>10</y>
          <width>2</width>
          <height>48</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="Line" name="lnDelimiter2">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>10</y>
          <width>2</width>
          <height>48</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QToolButton" name="tbZoomFix">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>10</y>
          <width>48</width>
          <height>48</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton
{
        border-image: url(:/imageviewerdlg/Icon/imageviewerdlg/zoom-fix.png);
	background:transparent;
	border: 2px solid lightgray;
	border-radius: 10px;
}

QToolButton::hover
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,  
                                 stop: 0 #758385, stop: 0.5 #122C39,  
                                 stop: 1.0 #0E7788);  

    border-color: #11505C;
}

QToolButton::pressed
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,  
                                 stop: 0 #969B9C, stop: 0.5 #16354B,  
                                 stop: 1.0 #244F76);

	border-color: #11505C;
}

QToolTip
{
	background: #FFFFA0;
	border: 0px solid green;
    border-radius: 0px;
    padding: 0px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/icon/zoom_fix.png</normaloff>:/icon/zoom_fix.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
       <widget class="QToolButton" name="tbZoomOriginal">
        <property name="geometry">
         <rect>
          <x>185</x>
          <y>10</y>
          <width>48</width>
          <height>48</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton
{
        border-image: url(:/imageviewerdlg/Icon/imageviewerdlg/zoom-original.png);
	background:transparent;
	border: 2px solid lightgray;
	border-radius: 10px;
}

QToolButton::hover
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,  
                                 stop: 0 #758385, stop: 0.5 #122C39,  
                                 stop: 1.0 #0E7788);  

    border-color: #11505C;
}

QToolButton::pressed
{
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,  
                                 stop: 0 #969B9C, stop: 0.5 #16354B,  
                                 stop: 1.0 #244F76);

	border-color: #11505C;
}

QToolTip
{
	background: #FFFFA0;
	border: 0px solid green;
    border-radius: 0px;
    padding: 0px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/icon/zoom_original.png</normaloff>:/icon/zoom_original.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
       <zorder>tbZoomOut</zorder>
       <zorder>tbZoomIn</zorder>
       <zorder>tbNavigationPrevious</zorder>
       <zorder>tbNavigationNext</zorder>
       <zorder>tbRotateLeft</zorder>
       <zorder>tbRotateRight</zorder>
       <zorder>tbFullScreenImage</zorder>
       <zorder>lnDelimiter4</zorder>
       <zorder>lnDelimiter3</zorder>
       <zorder>lnDelimiter1</zorder>
       <zorder>lnDelimiter2</zorder>
       <zorder>tbZoomFix</zorder>
       <zorder>tbZoomOriginal</zorder>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
